modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/20_mount.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+'require fs';
+'require rpc';
+
+var callMountPoints;
```

Comment:
You could delete this line and add var to this line.
https://github.com/openwrt/luci/pull/2767/files#diff-389d15d60f4efa0b48e4381ec566f58603780d102182d1c9071293a9be6c24bcR7